config/formats.js

```diff
@@ -401,38 +401,51 @@ let Formats = [
 		column: 2,
 	},
 	{
-		name: "[Gen 7] ZU",
-		desc: `The usage-based tier below PU.`,
+		name: "[Gen 7] Suicide Cup",
+		desc: `Victory is obtained when all of your Pok&eacute;mon have fainted.`,
 		threads: [
-			`&bullet; <a href="https://www.smogon.com/forums/threads/3629669/">ZU</a>`,
+			`&bullet; <a href="https://www.smogon.com/forums/threads/3633603/">Suicide Cup</a>`,
 		],
 
-		mod: 'gen7',
-		ruleset: ['[Gen 7] PU'],
+		mod: 'suicidecup',
+		forcedLevel: 100,
+		ruleset: ['[Gen 7] Ubers', '!Mega Rayquaza Clause', '!OHKO Clause'],
 		banlist: [
-			// PU
-			'Abomasnow', 'Absol', 'Aggron', 'Archeops', 'Articuno', 'Audino', 'Aurorus', 'Claydol', 'Clefairy', 'Drampa',
-			'Eelektross', 'Exeggutor-Alola', 'Ferroseed', 'Floatzel', 'Gastrodon', 'Golurk', 'Gourgeist-Super', 'Gurdurr',
-			'Haunter', 'Hitmonchan', 'Kabutops', 'Kangaskhan', 'Kecleon', 'Kingler', 'Komala', 'Lanturn', 'Lilligant',
-			'Mesprit', 'Mudsdale', 'Oricorio-Sensu', 'Passimian', 'Persian-Alola', 'Primeape', 'Pyroar', 'Raichu-Alola',
-			'Regirock', 'Sableye', 'Sandslash-Alola', 'Silvally-Fairy', 'Silvally-Ghost', 'Skuntank', 'Spiritomb', 'Swanna',
-			'Stoutland', 'Togedemaru', 'Ursaring', 'Weezing', 'Zangoose',
-			// ZUBL
-			'Carracosta', 'Crabominable', 'Exeggutor', 'Gorebyss', 'Jynx', 'Ludicolo', 'Musharna', 'Raticate-Alola',
-			'Raticate-Alola-Totem', 'Throh', 'Turtonator', 'Type: Null', 'Victreebel',
+			'Shedinja', 'Infiltrator', 'Magic Guard', 'Misty Surge', 'Assault Vest', 'Explosion', 'Final Gambit', 'Healing Wish',
+			'Lunar Dance', 'Magic Room', 'Memento', 'Misty Terrain', 'Self-Destruct',
 		],
+		unbanlist: ['Baton Pass'],
+		onValidateTeam: function (team) {
+			if (team.length !== 6) return [`Your team cannot have less than 6 Pok\u00e9mon.`];
+		},
 	},
 	{
-		name: "[Gen 7] Inverse",
-		desc: `The type effectiveness chart is inverted.`,
+		name: "[Gen 7] Scalemons",
+		desc: `Every Pok&eacute;mon's stats, barring HP, are scaled to give them a BST as close to 600 as possible.`,
 		threads: [
-			`&bullet; <a href="https://www.smogon.com/forums/threads/3590154/">Inverse</a>`,
+			`&bullet; <a href="https://www.smogon.com/forums/threads/3607934/">Scalemons</a>`,
 		],
 
 		mod: 'gen7',
-		ruleset: ['[Gen 7] OU', 'Inverse Mod'],
-		banlist: ['Diggersby', 'Hoopa-Unbound', 'Kartana', 'Kyurem-Black', 'Serperior', 'Tapu Bulu', 'Tapu Lele'],
-		unbanlist: ['Aegislash', 'Dialga', 'Giratina', 'Lucario-Mega', 'Solgaleo'],
+		ruleset: ['[Gen 7] Ubers', '!Mega Rayquaza Clause'],
+		banlist: [
+			'Gengar-Mega', 'Mawile-Mega', 'Medicham-Mega', 'Shedinja', 'Arena Trap', 'Shadow Tag', 'Deep Sea Scale',
+			'Deep Sea Tooth', 'Eevium Z', 'Eviolite', 'Light Ball', 'Thick Club',
+		],
+		onModifyTemplate: function (template, target, source) {
+			if (source) return;
```

Comment:
@MacChaeger's changes changed this, I think; you might need `if (!source) return;` instead. See #4688.